modules/shapes/module.php

```diff
@@ -8,6 +8,29 @@
 
 class Module extends \Elementor\Core\Base\Module {
 
+	public function __construct() {
+		parent::__construct();
+
+		add_action( 'wp_enqueue_scripts', [ $this, 'register_style' ] );
+	}
+
+	/**
+	 * Register styles.
+	 *
+	 * At build time, Elementor compiles `/modules/shapes/assets/scss/frontend.scss`
+	 * to `/assets/css/widget-shapes.min.css`.
+	 *
+	 * @return void
+	 */
+	public function register_style() {
+		wp_register_style(
+			'widget-shapes',
+			$this->get_css_assets_url( 'widget-shapes', null, true, true ),
+			[],
+			ELEMENTOR_VERSION
+		);
+	}
```

Comment:
Yeah, I think we need a similar method for these modules with widgets:

https://github.com/elementor/elementor/pull/28128/files#diff-25f13c7a23bc088a3c138855fa91482dd8a779a109f0adfea34208a36c81d956R563-R572